streams/vn_vthanhtivi.m3u

```diff
@@ -0,0 +1,11 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="ONEHD.sg",One HD
+https://code.vthanhtivi.pw/getlink/eazie/243114/playlist.m3u8
+#EXTINF:-1 tvg-id="",Cinemax Asia (Vietnam)
+http://fpt.vthanhtivi.pw/cinemax/index.m3u8
+#EXTINF:-1 tvg-id="",HBO (Vietnam)
+http://fpt.vthanhtivi.pw/hbo/index.m3u8
+#EXTINF:-1 tvg-id="GEM.sg",GEM
+https://code.vthanhtivi.pw/getlink/eazie/242918/playlist.m3u8
+#EXTINF:-1 tvg-id="",History (Vietnam)
+http://fpt.vthanhtivi.pw/history/mpegts
```

Comment:
Wouldn't it be easier to use the index.m3u8 file ? http://fpt.vthanhtivi.pw/history/index.m3u8